tools/ms2snoop/MS2snoop.xml

```diff
@@ -1,117 +1,215 @@
-        />
-        <param
-            name="min_number_scan"
-            type="integer"
-            min="0"
-            max="25"
-            value="8"
-            label="Present in at least X scan"
-            help="fragments are kept if they are found in a minimum number of scans"
-        />
+
+        <conditional name="param">
+
+            <param
+                type="boolean"
+                value="false"
+                label="Prints version"
+                argument="--version"
+                help="The tool prints its version and then exits"
+            />
+
+            <when value="false">
+
+                <param
+                    type="data"
+                    format="tabular"
+                    name="compound_txt"
+                    label="compounds_pos.txt"
+                />
+                <param
+                    type="data"
+                    format="tabular"
+                    name="peaklist_frag_tsv"
+                    label="peaklist_fragments.tsv"
+                />
+                <param
+                    type="data"
+                    format="tabular"
+                    name="peaklist_preco_tsv"
+                    label="peaklist_precursors.tsv"
+                />
+                <param
+                    type="float"
+                    min="0.0001"
+                    max="10"
+                    value="0.01"
+                    label="MZ Tolerence"
+                    argument="--tolmz"
+                />
+                <param
+                    type="integer"
+                    min="0"
+                    max="30"
+                    value="20"
+                    label="RT Tolerence"
+                    argument="--tolrt"
+                />
+                <param
+                    type="float"
+                    min="0"
+                    max="1"
+                    value="0.05"
+                    label="r pearson correlation threshold between precursor and fragment absolute intensity"
+                    argument="--seuil_ra"
+                />
+                <param
+                    type="integer"
+                    min="0"
+                    max="5"
+                    value="0"
+                    label="nb decimal for mz"
+                    argument="--mzdecimal"
+                />
+                <param
+                    type="float"
+                    min="0"
+                    value="0.85"
+                    label="fragment absolute intensity"
+                    argument="--r_threshold"
+                />
+                <param
+                    type="integer"
+                    min="0"
+                    max="25"
+                    value="8"
+                    label="Present in at least X scan"
+                    argument="--min_number_scan"
+                    help="fragments are kept if they are found in a minimum number of scans"
+                />
+                <param
```

Comment:
verbose and debug I would not provide as a user choice, or at least hidden into an advanced section

Reply:
fixed